Align "UA" stylesheet for "marquee" with HTML Spec

Ahmad-S792 opened the pull request:
#### eedc1e9585beba06191262f18c532432ad2e23c5
<pre>
Align &quot;UA&quot; stylesheet for &quot;marquee&quot; with HTML Spec

Align &quot;UA&quot; stylesheet for &quot;marquee&quot; with HTML Spec

<a href="https://bugs.webkit.org/show_bug.cgi?id=245160">https://bugs.webkit.org/show_bug.cgi?id=245160</a>

Reviewed by Tim Nguyen.

Align &quot;UA&quot; stylesheet with latest HTML Specifications:

- marquee element is missing &quot;text-align: initial&quot;.

<a href="https://html.spec.whatwg.org/#the-marquee-element-2">https://html.spec.whatwg.org/#the-marquee-element-2</a>

* Source/WebCore/css/html.css - Update &quot;UA&quot; stylesheet with HTML Specifications

Canonical link: <a href="https://commits.webkit.org/254486@main">https://commits.webkit.org/254486@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/457db32e9627d6b2a425ff9830f60bebb15d08ab

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/89194 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/33755 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/20014 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/98513 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/154826 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/93203 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/32263 "Built successfully") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/27811 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/81556 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/92982 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/94841 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/25629 "Passed tests") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/76119 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/25556 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/80494 "Passed tests") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/80467 "Build is in progress. Recent messages:Running worker_preparation") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/68530 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/30042 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/14530 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/29766 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/70/builds/15493 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/3146 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/33213 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/76119 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/31914 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/62/builds/34584 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
looks good, please update the commit message from:
```
*Source/WebCore/css/html.css - Update "UA" stylesheet with HTML Specifications
```
to
```
* Source/WebCore/css/html.css: Update "UA" stylesheet with HTML Specifications
```
for consistency with other ones

Commit:
Align "UA" stylesheet for "marquee" with HTML Spec

Align "UA" stylesheet for "marquee" with HTML Spec

https://bugs.webkit.org/show_bug.cgi?id=245160

Reviewed by Tim Nguyen.

Align "UA" stylesheet with latest HTML Specifications:

- marquee element is missing "text-align: initial".

https://html.spec.whatwg.org/#the-marquee-element-2

* Source/WebCore/css/html.css - Update "UA" stylesheet with HTML Specifications

Canonical link: https://commits.webkit.org/254486@main
